app/models/damage.rb

```diff
@@ -0,0 +1,24 @@
+class Damage < ActiveRecord::Base
```

Comment:
Because making a reservation can let the item automatically become available after the repair reservation period.